file_renamer.py

```diff
@@ -4,12 +4,30 @@
 import sys
 import json
 from datetime import datetime
+import argparse
 
-target_path = ''
+# Define the parser
+parser = argparse.ArgumentParser(description='parse arguments from cli')
 
-# substring/regex replacements rules will be built here
-replacements = {}
+parser.add_argument('--target-path', action="store", dest='target_path', default="null")
+parser.add_argument('--name', action="store", dest='name', default="null")
+parser.add_argument('--replacement', action="store", dest='replacement', default="null")
+args = parser.parse_args()
+target_path = args.target_path
+name = args.name
```

Comment:
can you rename this variable to "string" so it's clear to a user that it doesn't have to be the entire filename, but can just be a part of a filename they want to be replaced

Reply:
I suppose I could do this for you good sir